and_scale_ros/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package>
+  <name>and_scale_ros</name>
+  <version>0.0.0</version>
+  <description>The and_scale_ros package</description>
+
+  <maintainer email="pazeshun3682@gmail.com">Shun Hasegawa</maintainer>
+  <author email="pazeshun3682@gmail.com">Shun Hasegawa</author>
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <run_depend>jsk_recognition_msgs</run_depend>
```

Comment:
if one of packages in jsk_recognition repository depends on some of the jsk_3rdparty package, then we'll have trouble on releasing, specially when we move to new distro.
for example, when you update jsk_3rdparty and travis need binary version of jsk_recognition, we have to release jsk_recognition before that, I do not think that's going to happen.